### PR TITLE
Add Python code example for taskToDomain feature

### PR DESCRIPTION
**Pull request by supriyanta**
Screenshot 2023-07-07 at 12 34 53 AM

### CODE REVIEW

File: docs/developer-guides/task-to-domain.md [hunk 1]
```diff
@@ -87,7 +87,62 @@ View the [complete code here](https://github.com/orkes-io/orkes-conductor-client
...
+        server_api_url='https://play.orkes.io/api',
+        debug=True
+    )
+
+    workers = [
+        Worker(
+            task_definition_name='task_1',
+            execute_function=execute,
+            poll_interval=0.25,
+            domain="test_function" # specifying DOMAIN for function workers
```
Comment (coderabhigupta) [inline docs/developer-guides/task-to-domain.md:134]: Let's call this "test" only to be consistent with the documentation above.

File: docs/developer-guides/task-to-domain.md [hunk 2]
```diff
@@ -87,7 +87,62 @@ View the [complete code here](https://github.com/orkes-io/orkes-conductor-client
...
+        ),
+        SimpleWorker(task_definition_name="task_2")
+    ]
+
+    # Starting the Workers
+    with TaskHandler(workers, configuration) as task_handler:
+        task_handler.start_processes()
+        task_handler.join_processes()
```
Comment (coderabhigupta) [inline docs/developer-guides/task-to-domain.md:142]: We can remove this as well because we're already linking the code.

Comment (supriyanta) [reply docs/developer-guides/task-to-domain.md]: Removed this part @coderabhigupta

File: docs/developer-guides/task-to-domain.md (hunk 1 above)
Comment (supriyanta) [reply docs/developer-guides/task-to-domain.md]: Made the changes @coderabhigupta

**Review by coderabhigupta — APPROVED** [review]

**Pull request merged**